### DISCUSSION TITLE
Requesting guidance on how to open an issue.

### DISCUSSION THREAD

**Discussion by nidalhaddad** [i3/i3#4831] [Q&A]
Hi, i am requestion guidance on how to open an issue regarding slack application. 
My problems with slack are the following: 
1- Slack icone showing when using `rofi -show drun` but not showing when using `rofi -show window` to show active windows 
![slack](https://user-images.githubusercontent.com/97884446/152145369-adfd79a1-11e0-4e1a-88b1-8f5580af46bf.png)

2- Slack and other applications create a child window in the same workspace but cant close them using key combination ( ctrl+q nor ctrl+w ) to close them. i need to close the parent for it to close. 

What files should i put in my issue so someone can check it out  ?
Thank you.

**Comment by orestisfl** [comment]
It's not clear to me why an issue should open in i3 instead of rofi

**Reply by nidalhaddad** [reply]
well maybe the first part is a rofi issue, you are correct. but how about the second issue ? where the more than one application, ( slack, draw.io and teams ) create a child window in the same workspace and cant be killed just like the parent window ? ( in order to exit the child i have to terminate the parent before)

**Reply by orestisfl** [reply]
Is the key combination in your i3 config? You can start by sharing that.

**Reply by nidalhaddad** [reply]
yes it is. i have 2 ways to kill an application. `ctrl+q `to quit or` $mod+Shift+A kill` and none of them work on the child.
